src/jira.py

```diff
@@ -0,0 +1,144 @@
+import base64
+import re
+import requests
+import pprint
+import time
+
+from src.access import get_access_params
+from src.board import Issue
```

Comment:
If I don't import `Issue`, don't inherit and initiate from it, `test_jira` still passes. Either relevant lines in `board.py` are not covered by the test, or they are not executed. Please investigate and change accordingly.

Reply:
added tests for Issue methods